### PR TITLE
There are some corrections and I organized everything.I run everythin…

### PR DESCRIPTION
**Pull request by LefterisMal**
@haslbeck 
I made a readme file with instrunctions for each case we have to run and I executed the code that generates all the config files.I only have to add the last simulation folder (Scintillator_shape) that changes the detector's shapes (I am almost done) with some instrunctios on how to use the detectors parameters.

### CODE REVIEW

File: simulation/Scintillator_Energy/example/energy_simulation_example1.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+E_min=1
+E_max=50
+
+#Number of possible energy values(this depends on SiPM's accuracy):
+N_E=6
+
+#For a random sample define 1,for a perfect CM distribution 0:
+random=1
+
+def MetropolisHastings(pdf, N, x_init, sigma):
```
Comment (haslbeck) [inline simulation/Scintillator_Energy/example/energy_simulation_example1.py:22]: ahah nice one! -fh

File: simulation/Scintillator_Energy/example/energy_simulation_example1.py [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+      chain_points[i,:]=x_old
+
+    return chain_points
+    
+def pdf_theta(x,theta_max=theta_max*np.pi/180):
+    if (x>theta_max) or (x<0):
+        return 0
+    return np.cos(x)**2
+
+def norm(theta,E_min=E_min,E_max=E_max,N_E=N_E):
```
Comment (haslbeck) [inline simulation/Scintillator_Energy/example/energy_simulation_example1.py:41]: what is happening here -fh

File: simulation/Scintillator_Energy/example/energy_simulation_example1.py [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+    dE=(E_max-E_min)/(N_E-1)
+    e_k=115
+    e_p=850
+    E=E_min
+    sum=0
+    for i in range (N_E):
+        sum+=0.14*dE*(E**(-2.7))*(((1/(1+1.1*E*np.cos(theta)/e_p))+(0.054/(1+1.1*E*np.cos(theta)/e_k))))
```
Comment (haslbeck) [inline simulation/Scintillator_Energy/example/energy_simulation_example1.py:48]: where do the values come from? -fh

File: simulation/Scintillator_Energy/example/energy_simulation_example1.py [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+   directionz=np.zeros((n_theta,n_phi))
+   theta=np.linspace(0,theta_max,n_theta)
+   phi=np.linspace(0,2*np.pi-2*np.pi/n_phi,n_phi)
+
+   for i in range (n_theta):
+       R=z[0,0]*np.tan(theta[i])
+       for j in range (n_phi):
+           x[i,j]=R*np.cos(phi[j])
+           y[i,j]=R*np.sin(phi[j])
+           directionx[i,j]=-np.sin(theta[i])*np.cos(phi[j])
```
Comment (haslbeck) [inline simulation/Scintillator_Energy/example/energy_simulation_example1.py:80]: can u add some comments? why =- and not -=?

File: simulation/Scintillator_Energy/readme.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+##E_min and E_max Simulation:
```
Comment (haslbeck) [inline simulation/Scintillator_Energy/readme.md:1]: cool! -fh

File: simulation/Scintillator_Layer_Detector/scintillator_detector.conf [hunk 6]
```diff
@@ -0,0 +1,156 @@
+
```
Comment (haslbeck) [inline simulation/Scintillator_Layer_Detector/scintillator_detector.conf:1]: cool! is there a way we can visualize this geometry once the build succeeds? -fh

**Review by haslbeck — APPROVED** [review]
nice work! could you add some comments on the .py script (See my comments)? Its hard to follow whats going on and why:D @LefterisMal -fh

File: simulation/Scintillator_Energy/example/energy_simulation_example1.py (hunk 3 above)
Comment (LefterisMal) [reply simulation/Scintillator_Energy/example/energy_simulation_example1.py:48]: I will add some theoretical stuff on slack on the theoretical stuff and formula's.I can add links for those in the readme

File: simulation/Scintillator_Energy/example/energy_simulation_example1.py (hunk 4 above)
Comment (LefterisMal) [reply simulation/Scintillator_Energy/example/energy_simulation_example1.py:80]: Yeah,I am going to send those to the others to check if they understand the whole process and if we need to add more details.The - in the direction comes from the fact that the source point position is always above the layers and target the (0,0,h/2).I will add a notebook that visualise the first functions,I think that's the easiest way to understand everything

File: simulation/Scintillator_Layer_Detector/scintillator_detector.conf (hunk 6 above)
Comment (LefterisMal) [reply simulation/Scintillator_Layer_Detector/scintillator_detector.conf:1]: Yes,the model we are based on is all for visualisation by Geant4(and this is also our main problem,I am trying to check if it's possible for a start to run this without visualisation).But we will need the visualisation to check the dimensions and sensor's position, because this model is a project outside the main code I am not 100% sure if all the formalism details are the completely same.
